cmd/supernode/app/root_test.go

```diff
@@ -49,7 +49,7 @@ func (s *SupernodeAppTest) SetUpSuite(c *check.C) {
 func (s *SupernodeAppTest) TearDownSuite(c *check.C) {
 	if s.workHome != "" {
 		if err := os.RemoveAll(s.workHome); err != nil {
-			fmt.Printf("remove path:%s error", s.workHome)
+			printer.Printf("remove path:%s error", s.workHome)
```

Comment:
What's your purpose of updating it?

Reply:
taiyun tell me to avoid to use fmt and use printer.i  have modify it in previous pr .

Reply:
It's unnecessary to use it in unit testing.

Reply:
> It's unnecessary to use it in unit testing.

get